Sources/SentryDistribution/Models/CheckForUpdateParams.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A model for configuring parameters needed to check for app updates.
+public struct CheckForUpdateParams {
```

Comment:
`m`: It can easily happen that the default values are changed by accident. Please add unit tests so we can detect unwanted changes